napa.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'virtus', '~> 1.0.0'
   gem.add_dependency 'grape', '~> 0.10.0'
   gem.add_dependency 'grape-swagger', '~> 0.9.0'
-  gem.add_dependency 'roar', '~> 0.12.0'
+  gem.add_dependency 'roar', '>= 0.12.0'
```

Comment:
This could use a specific pessimistic approach.

So we could do `gem.add_dependency 'roar', '>= 0.12.0', '< 2.0.0'`